### ISSUE TITLE
LoL in Edge creates dotted line in Multiple Cells

### ISSUE THREAD

**Issue by BryceAG** [phetsims/gene-expression-essentials#110]
there seems to be a slight delay throughout all of Edge when pausing, but this only causes an issue when looking at the graph in Multiple Cells. With many cells active, if paused and then resumed it will create dots as shown here:
![image](https://user-images.githubusercontent.com/15327201/32917572-d389e4fa-cadc-11e7-940b-fe7ee6a4d5eb.png)

It is important to note that it must be paused for about a second at least, as rapid pausing and resuming will still have a continuous line

related to phetsims/QA#68

**Comment by jbphet** [comment]
This turned out to be a combination of two issues.  The first one was quite subtle - it was a problem where the data line was **always** being drawn with dashes, but since the data points were so close together, it generally didn't show up.  When restarting after a long pause, the gap in time allowed the gaps in the line to become visible.  The gaps in the line were due to some oddness about the rendering, and I logged a Scenery issue about it in https://github.com/phetsims/scenery/issues/717.  I added a workaround.

The second issue is that the time step - dt - was not being correctly limited on this screen.  I fixed that as well.

The problem should be resolved on both master and the release branch, and I'll leave this issue open and will have it tested on the next release candidate.
